new_client/src/components/Footer.css

```diff
@@ -1,46 +1,15 @@
 .footer {
-    float: left;
-    width: 100%;
-    background-color: #000000;
-    overflow: auto;
-    padding-left: 40px;
-  
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 2%;
```

Comment:
As shown in the photos show to you if we are keeping footer on all pages then we still need to align it